plugin/storage/es/options.go

```diff
@@ -313,60 +369,80 @@ func (opt *Options) InitFromViper(v *viper.Viper) {
 }
 
 func initFromViper(cfg *namespaceConfig, v *viper.Viper) {
-	cfg.Username = v.GetString(cfg.namespace + suffixUsername)
```

Comment:
could you please undo this `flagWithSuffix` change? It's superficial, doesn't make the core easier to read (actually has longer text) and creates too much noise in the PR that's not relevant to the main change

Reply:
ok